internal/multistorage/cache/file.go

```diff
@@ -11,7 +11,7 @@ import (
 )
 
 // StatusFile is stored on a disk and therefore shared between all WAL-G processes and commands.
-const StatusFile = ".walg_storage_status_cache"
+const StatusFile = "/tmp/.walg_storage_status_cache"
```

Comment:
I see three options where to store this cache file:
1. In a user folder, as previously. @usernamedt can you please explain why you decided to put it there
2. In a working directory. I made it so initially, but in our Docker tests the working directory is "/" so the user doesn't have access to it
3. In "/tmp" directory, every user should have access to it so that the cache will be shared

We can also add a setting to the config file, because there may be a problem if two different WAL-G configs are used on the same machine but they use the same name for different failover storages

Reply:
We've decided to stick with the per-user-folder approach. It would automatically avoid any permission errors, also it would stay consistent with the [decompressor cache file](https://github.com/wal-g/wal-g/blob/ab22a2303e5ddedeabc3706b641ec97be9e2974d/internal/fetch_helper.go#L112) logic.